config/environments/development.rb

```diff
@@ -100,14 +100,14 @@
   # If you are using HTTP's basic authentication, you probably want to use this
   # option.
 
-  LOGOUT_REDIRECT = 'DEFAULT'
+  LOGOUT_REDIRECT = "DEFAULT"
 
   ###################################################################
   # File storage (Repository) settings
   ###################################################################
   # Options for Repository_type are 'svn' and 'memory' for now
   # 'memory' is by design not persistent and only used for testing MarkUs
-  REPOSITORY_TYPE = 'git' # use Subversion as storage backend
+  REPOSITORY_TYPE = "svn" # use Subversion as storage backend
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.